python/py-catalogue/Portfile

```diff
@@ -59,5 +60,10 @@ if {${name} ne ${subport}} {
             ${destroot}${docdir}
     }
 
+     test.run        yes
+     test.cmd        py.test-${python.branch}
+     test.target
+     test.env        PYTHONPATH=${worksrcpath}/build/lib
+
```

Comment:
why duplicating this, it's already set above the `post-destroot` part it seems

Reply:
Oops, thanks, fixed.